src/components/tab-group/tab-group.spec.ts

```diff
@@ -0,0 +1,134 @@
+import {
+    it,
+    expect,
+    beforeEach,
+    inject,
+    describe
+} from '@angular/core/testing';
+import {TestComponentBuilder, ComponentFixture} from '@angular/compiler/testing';
+import {MD_TAB_GROUP_DIRECTIVES, MdTabGroup} from './tab-group';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+
+describe('MdTabGroup', () => {
+  let builder: TestComponentBuilder;
+
+  beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
+    builder = tcb;
+  }));
+
+  it('should default to the first tab', (done: () => void) => {
```

Comment:
Rather than passing in `done` and calling it manually, you can now use `async` when dealing with `createAsync`, e.g.:
```ts
it('should do something', async(() => {
  builder.createAsync(TestApp).then(fixture => {
    expect(1).toBe(1);
  });
}));
```
`async()` hooks into zones and makes it so that the test doesn't "complete" until all of the microtasks have been executed.